app/js/components/LabOrdersList/LabOrdersList.jsx

```diff
@@ -86,26 +86,32 @@ export class LabOrdersList extends PureComponent {
       this.handlePatientChange();
     }
 
-    // if a specific order uuid has been specified, redirect to the display
-    // page for that order after the orders have been fetched
-    if (fetched && !prevProps.fetched && match.params.orderUuid && orders) {
+    // if a specific order uuid (or set of order uuids) has been specified, redirect to the display
+    // page for first order after the orders have been fetched; set a "afterSubmitLink" with the
+    // remaining order ids so we cycle through them; this provides a way for our Order Entry app to
+    // redirect to the entry pages for a set of orders after placing the order
+    if (fetched && !prevProps.fetched && match.params.orderUuids && orders) {
+      const [orderUuid, ...otherOrderUuids] = match.params.orderUuids.split(",");
```

Comment:
Interesting syntax - this works?

Reply:
Yep, the spread & rest operators and destructing are the best practice way (I think?) of doing this kind of stuff in ESM6

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax